client/js/analysisui/formatdef.js

```diff
@@ -0,0 +1,218 @@
+
+'use strict';
+
+var _ = require('underscore');
+
+var Formatdef = {
```

Comment:
wasn't sure. i kind of use it as a static class. So are you suggesting formatDef or FormatDef? I feel like the second.